tICA/README.md

```diff
@@ -0,0 +1,19 @@
+# Temporal ICA
+
+This pipeline uses ICA on a large amount of fMRI data (typically a large group of subjects),
+in order to separate global nuisance effects (such as due to respiration) from neural signals that
+have a nonzero spatial mean.
+
+The processing modes also allow for applying temporal ICA cleanup to datasets that are not
+large enough to reliably produce a new temporal ICA decomposition.  Briefly, the processing
+modes are:
+
+- NEW - do a full pipeline run from scratch, starting with a fresh MIGP, intended for large datasets
+
+- REUSE_SICA_ONLY - use a previous MIGP result, but estimate the temporal ICA from scratch
```

Comment:
So, presumably both NEW and REUSE_SICA_ONLY require users to classify (themselves) the tICA components?  Can we make that explicit, and elaborate on that process?

Reply:
We plan to add automated classification, but it isn't integrated yet.